package.json

```diff
@@ -63,6 +64,11 @@
     "bole": "^4.0.0",
     "bunyan": "^1.8.14",
     "docsify-cli": "^4.4.1",
+    "eslint": "^7.17.0",
+    "eslint-config-standard": "^16.0.2",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
```

Comment:
Are these necessary?

Reply:
Didn't know if you want extra funct. I will remove

Reply:
@jsumners I did an investigation and all of these 3 need to be in devDependencies - won't work without it, this package is dependent on them.